### PR TITLE
update sprint-schedule.md

### PR DESCRIPTION
**Pull request by amilash**
updated with our current meetings and reoccurring meetings

### Summary of Changes
_Provide a brief summary of changes_
_Addresses issue #_
_Acceptance criteria as stated in the issue_

### How to Test
_List the steps to test the PR_

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [ ] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [ ] Check against the criteria:

As facilitator/product manager, @kniz-raft will decide if ACs are met from Raft's perspective.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [ ] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Does this PR change any linting or CI settings?

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*

+ [ ] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

> NOTE: until we have a proper staging environment this may not be satisfiable prior to merging

+ [ ] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [ ] If this PR introduces backend code, is that code documented both inline and overall?
+ [ ] If this PR introduces frontend code, is that code documented both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any security issues?

### CODE REVIEW

File: docs/How-We-Work/team-charter/sprint-schedule.md [hunk 1]
```diff
@@ -2,61 +2,25 @@
...
+### Weekly Mondays
+- 11:00am-12:00pm ET: UX Sync - An as needed meeting between UX Design and Product for Design to discuss and demo research findings to Product and for Product to review/accept/reject new features and user stories into the Product Roadmap and version plan.
+
+### Biweekly on Tuesdays - Sprint End and Start: Planning Day!
+- 11:00am-12:00pm EST: Backlog Refinement 
+- 12:00pm-12:15pm EST: Standup
+- 12:15-12:3pm ET: Raft Demo Prep
+- 2:00pm-2:30pm ET: Sprint Demo
+- 2:30pm-3:00pm ET: Sprint Retrospective
+- Sprint goals and priorities are discussed in refinement and reviewed during sprint planning. The team uses these goals to prioritize work for the sprint.
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/sprint-schedule.md:18]: Should we add sprint planning here?

File: docs/How-We-Work/team-charter/sprint-schedule.md [hunk 2]
```diff
@@ -2,61 +2,25 @@
...
+
+### Weekly Wednesdays:
+- 12:00pm-12:15pm EST: Standup
+- 2:00pm-3:30pm ET: Dev Sync
+
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/sprint-schedule.md:23]: ### Biweekly on Wednesdays - mid-sprint
- 12:30pm-1:00pm EST: Integrated Project Team (IPT) w/ OCIO

File: docs/How-We-Work/team-charter/sprint-schedule.md [hunk 3]
```diff
@@ -2,61 +2,25 @@
...
+- 12:00pm-12:15pm ET: Daily Standup
+- If teammembers cannot make the team standup on zoom, an asynchronous standup update shall be given in the teams Microsoft Teams channel. On Thursdays, the team agrees to hold an asynchronous standup and forego the zoom call.
+
+### Weekly Mondays
+- 11:00am-12:00pm ET: UX Sync - An as needed meeting between UX Design and Product for Design to discuss and demo research findings to Product and for Product to review/accept/reject new features and user stories into the Product Roadmap and version plan.
+
+### Biweekly on Tuesdays - Sprint End and Start: Planning Day!
+- 11:00am-12:00pm EST: Backlog Refinement 
+- 12:00pm-12:15pm EST: Standup
+- 12:15-12:3pm ET: Raft Demo Prep
```
Comment (lfrohlich) [inline docs/How-We-Work/team-charter/sprint-schedule.md:15]:
```suggestion
- 12:15-12:30pm ET: Raft Demo Prep
```

File: docs/How-We-Work/team-charter/sprint-schedule.md (hunk 1 above)
Comment (amilash) [reply docs/How-We-Work/team-charter/sprint-schedule.md:19]: added

File: docs/How-We-Work/team-charter/sprint-schedule.md (hunk 2 above)
Comment (amilash) [reply docs/How-We-Work/team-charter/sprint-schedule.md:27]: added

**Review by amilash — COMMENTED** [review]
updated

**Review by amilash — COMMENTED** [review]
@lfrohlich  ready for re-review

**Review by lfrohlich — APPROVED** [review]
👍

**Pull request merged**